include/waypoints.h

```diff
@@ -10,57 +10,65 @@ typedef struct {
     /* 0x06 */ u16 trackSectionId;
 } TrackWaypoint; // size = 0x08
 
+enum {
+    RIGHT_LEANING_CURVE = 0,
+    LEFT_LEANING_CURVE = 1,
+    RIGHT_CURVE = 2, // strong curve
+    LEFT_CURVE = 3,  // strong curve
+    STRAIGHT = 4
+};
+
 /**
  * These are per-path arrays that contain some information relating to waypoints
- * The arrays in D_80164550 contain X/Y/Z and track segment information
- * The arrays in D_80164560 and D_80164570 track some other X/Y/Z, but the track segment is always 0 (so,
- *untracked/unused) Its unclear how these arrays relate to each other
+ * The arrays in gTrackPath contain X/Y/Z and track segment information
+ * The arrays in gTrackInnerPath and gCurrentTrackOuterPath track some other X/Y/Z, but the track segment is always 0
+ *(so, untracked/unused) Its unclear how these arrays relate to each other
  **/
-extern TrackWaypoint* D_80164550[];
-extern TrackWaypoint* D_80164560[];
-extern TrackWaypoint* D_80164570[];
+extern TrackWaypoint* gTrackPath[];
+extern TrackWaypoint* gTrackInnerPath[];
+extern TrackWaypoint* gTrackOuterPath[];
```

Comment:
I don't believe that gPaths[4] is equivallent to TrackPath, InnerPath, and Outter Path.

I think it's more like mainPath, and then other paths if they're used.
Most courses have 2 paths, I'm not super sure on the use of the second.